includes/admin/downloads/metabox.php

```diff
@@ -827,8 +845,14 @@ function edd_render_file_row( $key = '', $args = array(), $post_id, $index ) {
 			<input type="hidden" name="edd_download_files[<?php echo $key; ?>][index]" class="edd_repeatable_index" value="<?php echo $index; ?>"/>
 		</span>
 		<span class="edd-repeatable-row-actions">
-			<a class="edd-remove-row edd-delete" data-type="file"><?php printf( __( 'Remove', 'easy-digital-downloads' ), $key ); ?><span class="screen-reader-text"><?php printf( __( 'Remove file %s', 'easy-digital-downloads' ), $key ); ?></span>
-			</a>
+		<?php
+		printf(
+			'<a aria-label="%s" class="edd-remove-row edd-delete" data-type="file">%s</a>',
+			/* translators: %s: download file */
+			sprintf( esc_html__( 'Remove file %s', 'easy-digital-downloads' ), $key ),
```

Comment:
```suggestion
			esc_attr( sprintf( __( 'Remove file %s', 'easy-digital-downloads' ), $key ) ),
```

We need to ensure we escape the replaced value as well.